chapters/textures.txt

```diff
@@ -839,33 +839,16 @@ sparse unbound texel is undefined:.
 If the image view has a depth/stencil format, the depth component is
 selected by the pname:aspectMask, and the operation is an code:OpImage*Dref*
 instruction, a depth comparison is performed.
-The value of the result [eq]#D# is [eq]#1.0# if the result of the compare
-operation is [eq]#true#, and [eq]#0.0# otherwise.
-The compare operation is selected by the pname:compareOp member of the
-sampler.
-
-[latexmath]
-+++++++++++++++++++
-\begin{aligned}
-D & = 1.0 &
-  \begin{cases}
-    D_{\textit{ref}} \leq D_{\textit{tex}} & \text{for LEQUAL}   \\
-    D_{\textit{ref}} \geq D_{\textit{tex}} & \text{for GEQUAL}   \\
-    D_{\textit{ref}} < D_{\textit{tex}}    & \text{for LESS}     \\
-    D_{\textit{ref}} > D_{\textit{tex}}    & \text{for GREATER}  \\
-    D_{\textit{ref}} = D_{\textit{tex}}    & \text{for EQUAL}    \\
-    D_{\textit{ref}} \neq D_{\textit{tex}} & \text{for NOTEQUAL} \\
-    \textit{true}                          & \text{for ALWAYS}   \\
-    \textit{false}                         & \text{for NEVER}
-  \end{cases} \\
-D & = 0.0 & \text{otherwise}
-\end{aligned}
-+++++++++++++++++++
-
-where [eq]#D~tex~# is the texel depth value and [eq]#D~ref~# is the
-reference value from the SPIR-V operand.
-If the image being sampled has a fixed-point format then the reference value
-is clamped to [0, 1] before the comparison operation.
+The value of the result, [eq]#D#, is [eq]#1.0# if the result of the
+comparison is [eq]#true#, and [eq]#0.0# otherwise.
+The compare operation is selected by the elink:VkCompareOp value set by
+slink:VkSamplerCreateInfo::pname:compareOp.
+The _reference_ and _target_ arguments in the operation specified by the
+elink:VkCompareOp are the texel depth value [eq]#D~tex~# and the reference
+value from the SPIR-V operand [eq]#D~ref~#, respectively.
+
+If the image being sampled has a fixed-point format, then [eq]#D~ref~# is
```

Comment:
Should precisely say unsigned normalized fixed-point format.

Reply:
We do only have unsigned normalised depth formats in the spec, but what's the benefit of restating it here? The sentence seems true regardless, and if in some future extension we added an SNORM or SCALED format this would just be an interaction we're bound to miss.

Reply:
Purpose is linguistical precision and less brittleness. As you point out, if you added SCALED, this interaction is bound to be missed.

If it is signed, then it makes less sense for it to be clamped to 0. If it is not normalized, then it makes less sense to be clamped to unit. If you later added SCALED, this language would likely be defective. If you precisely specified the type, then it would "only" be unspecified for SNORM.

In a specification, being wrong is worse that being omitful. Ergo not being precise with the format type here is brittle.

Reply:
PS: Even less brittle or systematic would perhaps be to specify that the float32 is converted as per the conversion chapter (which already does include clamping).

Reply:
Fair point, I read the interaction differently but you're right, this really should only apply to those cases.